glue/core/cli_parsers/__init__.py

```diff
@@ -0,0 +1 @@
+from .parsers import parse_data
```

Comment:
```suggestion
from .parsers import parse_data  # noqa: F401
```
need a codestyle protection here?

Reply:
Thanks! I just pushed a commit I had locally that already fixed this.